checkov/secrets/coordinator.py

```diff
@@ -1,26 +1,24 @@
-from typing import Iterable, Dict, Optional
-
-
-class EnrichedSecret:
```

Comment:
can you instead create a `TypedDict` and then leverage it in `SecretsCoordinator` 🙂 then consumers will benefit from typing checks.